app/controllers/application_controller.rb

```diff
@@ -80,13 +80,24 @@ def authenticate_actions_for_role(role, opts={})
       course = opts[:course]
   
       # Get current user's role
-      current_role = course.membership_for_user(@user).role
+      membership = course.membership_for_user(@user)
```

Comment:
Changed all of the role variables in all files the same way?  Do you have documentation for why you changed the role?  (Just asking for future devs).

Reply:
Nope, I just got membership at this line to avoid Nil Exception.